packages/cf-style-const/src/variables.js

```diff
@@ -97,5 +97,5 @@ export default {
   colorFacebookBlue: '#3b5998',
   colorMainBackground: '#e6e6e6',
   zIndexMax: 1000,
-  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
+  modalBackdropBackground: 'rgba(0, 0, 0, .7)'
```

Comment:
Hey no worries! This is for a backdrop that covers the entire page. So it's a completely different pattern

Applied to a gh comment
```css
box-shadow: 0 1px 15px rgba(0,0,0,.75);
```

Cover entire page
<img width="1055" alt="screen shot 2017-06-06 at 2 44 28 pm" src="https://user-images.githubusercontent.com/1265379/26849521-2396f5c0-4aca-11e7-9591-c735cd356d2e.png">